bazel/repositories.bzl

```diff
@@ -220,7 +220,11 @@ def envoy_dependencies(skip_targets = []):
     external_http_archive("bazel_features")
     external_http_archive("bazel_toolchains")
     external_http_archive("bazel_compdb")
-    external_http_archive(name = "envoy_examples")
+    external_http_archive(
+        name = "envoy_examples",
+        patch_args = ["-p1"],
+        patches = ["@envoy//bazel:envoy_examples.patch"],
+    )
```

Comment:
you can create a PR to the envoy_examples directly and update the deps I think.

Reply:
https://github.com/envoyproxy/examples/pull/342